### PR TITLE
Changed protoc to build v3.13.0.1

### PR DESCRIPTION
**Pull request by stemann**
Needed for Torch v1.10.2

### CODE REVIEW

File: P/protoc/build_tarballs.jl [hunk 1]
```diff
@@ -3,11 +3,11 @@
 using BinaryBuilder, Pkg
 
 name = "protoc"
-version = v"3.13.0"
+version = v"3.13.0" # 3.13.0.1
```
Comment (giordano) [inline P/protoc/build_tarballs.jl:6]: How are you going to distinguish from the previous build?  Spoiler: you can't

Comment (stemann) [reply P/protoc/build_tarballs.jl:6]: I guess I would just be lucky to get 3.13.0+1 if I select v3.13.0 ? :-) ... and never be able to get v3.13.0...

Not sure what to do about those oddly named versions here...

Torch points to the 3.13.0.1 version (I was a bit fast to submit the other PR...).

Comment (giordano) [reply P/protoc/build_tarballs.jl:6]: I don't think the [difference between 3.13.0 and 3.13.0.1](https://github.com/protocolbuffers/protobuf/compare/v3.13.0...v3.13.0.1) is really significant for us.

Comment (stemann) [reply P/protoc/build_tarballs.jl:6]: Right - so let's close this PR?

Comment (giordano) [reply P/protoc/build_tarballs.jl:6]: Yeah, I don't think there is too much need for this

Comment (stemann) [reply P/protoc/build_tarballs.jl:6]: OK

**Pull request closed without merging**